python/desc/imsim/imSim.py

```diff
@@ -566,3 +566,13 @@ def get_logger(log_level):
                      eval('lsstLog.%s' % log_level))
 
     return logger
+
+
+def get_obs_lsstSim_camera(log_level=lsstLog.WARN):
+    """
+    Get the obs_lsstSim CameraMapper object, setting the default
+    log-level at WARN in ordert to silence the INFO message about
+    "Loading Posix exposure registry from ."
```

Comment:
Rather than saying this is to stop that particular info message, maybe we should just say we want to be able to set the log level?  The reason we got rid of it can be explained in this pull request but we might actually want to turn this up for debugging and someday that message might disappear in any case.

Reply:
If that's the approach you want to take, then we don't actually need the code in this PR.  The imsim.py executable already has a `--log_level` option, and if you set it to `WARN`, that info message is suppressed.  My feeling was that we want our `INFO` messages available, but that camera mapper message serves no useful purpose so we should just silence it, hence the comment in this function.

Reply:
Ah, OK.  OK I understand.

That's fine.  Maybe also mentioning in the comment that the overall level is at INFO would make this clearer.